themes/bootstrap3/js/combined-search.js

```diff
@@ -0,0 +1,22 @@
+/*global VuFind, setupOpenUrlLinks, checkSaveStatuses, setupSaveRecordLinks */
+VuFind.combinedSearch = (function() {
```

Comment:
I like it! Though in this instance, there doesn't seem to be any reason for the (VuFind) at the end, as the closure you've defined doesn't take any parameters. Based on the article, it seems that one approach might be to actually attach the combinedSearch logic to the VuFind object *inside* the closure, which would make use of the VuFind parameter. Alternatively, you could keep it as it is, but omit the unused parameter at the end.